src/Cassandra.IntegrationTests/TestClusterManagement/SimulacronManager.cs

```diff
@@ -9,31 +9,24 @@ public class SimulacronManager
     {
         private Process _simulacronProcess;
 
-        private static SimulacronManager _instance;
+        private bool _initialized;
 
-        private bool _started;
+        public static SimulacronManager Instance { get; } = new SimulacronManager();
 
-        public static SimulacronManager Instance
-        {
-            get
-            {
-                if (_instance != null) return _instance;
-                _instance = new SimulacronManager();
-                return _instance;
-            }
-        }
-
-        public static Uri BaseAddress
-        {
-            get { return new Uri("http://127.0.0.1:8188"); }
-        }
+        public static Uri BaseAddress => new Uri("http://127.0.0.1:8188");
 
         public void Start()
         {
-            if (_started)
+            lock (this)
```

Comment:
I think this is not necessary as we are not supporting calling `Start()` from multiple threads (otherwise we would have to use some kind of concurrency control to await until it was started).

Reply:
The integration tests are not run in parallel yet, but as it is something in radar, so I thought it would not hurt to put it already.